src/spotifun.tsx

```diff
@@ -16,11 +20,30 @@ export const Spotifun: React.FunctionComponent = (props => {
         }
     });
 
+    const onItemSelect = (selectedItem: React.MouseEvent) => {
```

Comment:
make this function of type (selectedItemId: string) =>void, and have the selectionBox component extract the id from the event

Reply:
fixed